src/index.js

```diff
@@ -241,4 +245,7 @@ export default (function create(/** @type {Options} */ defaults) {
 	redaxios.create = create;
 
 	return redaxios;
-})();
+};
+
+// @ts-ignore TS2554
+export default create();
```

Comment:
I tried changing L69 to `@param {Options} [default={}]` but TypeScript wasn't having it.

Reply:
fixed!